### ISSUE TITLE
(QUEST) Blacken the Skies

### ISSUE THREAD

**Issue by jfdaigle** [HellscreamWoW/Tracker#1530]
The "Wind rider" doesn't work.  You can kill Talrendis Glaive Thrower but can't Bomb the Command Center.  Can't complete.

http://www.wowhead.com/quest=24430/blacken-the-skies
https://www.youtube.com/watch?v=FI3i9v81L4Y&feature=youtu.be

**Comment by Malae1** [comment]
The vehicle part works though?

**Comment by jfdaigle** [comment]
No, there is no "Wind rider" to get on so you can't fly around and bomb.  What I ment is that you can kill the Talrendis Glaive Thrower "manually".

**Comment by Malae1** [comment]
Thanks.
